packages/peregrine/lib/talons/FilterModal/useFilterModal.js

```diff
@@ -123,11 +124,46 @@ export const useFilterModal = props => {
         }
     }, [filterKeys, filterState, history, isApplying, pathname, search]);
 
-    // on drawer toggle, read filter state from location
+    const handleOpen = useCallback(() => {
+        toggleDrawer(DRAWER_NAME);
+    }, [toggleDrawer]);
+
+    const handleClose = useCallback(() => {
+        closeDrawer(DRAWER_NAME);
```

Comment:
`closeDrawer` does not take an argument.
```suggestion
        closeDrawer();
```